python/mvdtool.cpp

```diff
@@ -177,6 +181,16 @@ PYBIND11_MODULE(mvdtool, mvd) {
                 const auto& func = [&f](const MVD::Range& r){return f.getMtypes(r);};
                 return _atIndices<std::string>(func, f.size(), idx);
              })
+        .def("layers", [](const File& f, int offset, int count) {
+                Range r(offset, count);
+                return f.getLayers(r);
+             },
+             "offset"_a = 0,
+             "count"_a = 0)
+        .def("layers", [](const File& f, pyarray<size_t> idx) {
+                const auto& func = [&f](const MVD::Range& r){return f.getLayers(r);};
+                return _atIndices<boost::int32_t>(func, f.size(), idx);
```

Comment:
I think we need to wrap this in a py::array, that's how I have seen it all around.
You can rebase on my version